CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+
+## Version 2.0.0-alpha.2
+
+- Font family and size selection in Preferences (Edit menu)
+- Highlight quotes in selection (Search menu)
+- Additional Compose Sequences added (Cmd+I/Ctrl+I) such as curly quotes, 
+  degrees, super/subscripts, fractions, Greek, etc. List of Sequences
+  (Help Menu) allows clicking to insert character. 
+
+### Bug Fixes
+
+- Ignore Case in Word Frequency stopped options such as ALL CAPS from working
+- Main window size and position didn't work well with maximized windows
+
+
+## Version 2.0.0-alpha.1
+
+First alpha release, containing the following features:
+- Status bar with line/col, Img, Prev/See/Next Img, Lbl, selection, language,
+  and current character display buttons
+- Go to line, image, label (click in status bar)
+- Restore previous selection (click in status bar)
+- Change language (click in status bar)
+- Line numbers (shift-click line/col in status bar)
+- Normal and column selection supported for most operations
+- Configure page labels (shift-click Lbl in status bar)
+- Built-in basic image viewer (View menu)
+- Auto Img (shift-click See Img in status bar)
+- Recent documents (File menu)
+- Change case features (Edit menu)
+- Preferences (Edit menu) for theme, font, margins, etc
+- Search & Replace (Search menu) with regex, match case, etc. Also, limit
+  search/replace to current selection
+- Count and Find All search features
+- Bookmarks (Search menu, and via shortcuts)
+- Basic Fixup (Tools menu)
+- Word Frequency (Tools menu)
+- Spell Check (Tools menu)
+- PPtxt  (Tools menu)
+- Jeebies (Tools menu)
+- Fixup Page Separators (Tools menu)
+- Fixup Footnotes (Tools menu) - begun but not complete
+- Rewrap (Tools menu) - using "greedy" algorithm
+- Unmatched Markup features  (Tools menu)
+- Convert Fractions (Tools menu)
+- Normalize Characters (Tools menu)
+- Add/Remove Page Marker Flags (File-->Project menu) for transfer between
+  editors
+- Compose Sequences (Tools menu, and via Cmd+I/Ctrl+I shortcut) - only
+  accented characters and Unicode ordinals. List of sequences (Help Menu)
+- Message log (View menu)
```

Comment:
Do you want to mention the `-d` option, or do you figure that's not something this group will be interested in? Are they still bringing it up from the command line?

Reply:
I don't think it's really needed, but I will mention it along with the -r option